sound/soc/sof/intel/hda-loader.c

```diff
@@ -310,13 +334,17 @@ int hda_cl_copy_fw(struct snd_sof_dev *sdev, struct hdac_ext_stream *hext_stream
 		dev_err(sdev->dev,
 			"%s: timeout with rom_status_reg (%#x) read\n",
 			__func__, chip->rom_status_reg);
+	} else {
+		dev_dbg(sdev->dev, "Code loader FW_ENTERED status\n");
```

Comment:
I would argue that `snd_sof_dsp_read_poll_timeout()` will also print out the value on success/fail.
Adding additional human readable message does not hurt, just pointing out.

Reply:
right, I just wanted a regular log that can be checked with grep, so that we can easily infer timing.